sqlalchemy_filters/filters.py

```diff
@@ -72,6 +73,29 @@ def format_for_sqlalchemy(self):
             return function(field, self.value)
 
 
+def filter_factory(filterdef, models):
+
+    if isinstance(filterdef, list):
+        return [filter_factory(item, models) for item in filterdef]
+
+    if isinstance(filterdef, dict) and {'or', 'OR'}.intersection(filterdef):
+        clauses = filterdef.get('or') or filterdef['OR']
+        # TODO raise error if clauses is not a list?
```

Comment:
Hi @juliotrigo - yep agree perhaps better to always have a list, and then just validate on the length of the list when handling the arguments to these functions